modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,41 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+These changes are automatically saved every few minutes while you are working on your project.
+The saved states are listed by time/date and activity, and their name depends on the activity you were doing at the time the save occurred.
+For example, configuring a trigger in a flow.
+
+Being able to view a saved state of a project is useful if you want to:
+
+* Detect what is or is not working between two different versions. For example, you started with a working, tested flow but during the configuration, you see errors and you want to revert back to the working state.
```

Comment:
"but during configuration" is confusing to me.  One option that is clearer to me is "...with a working, tested flow, but after changes were made, you see errors..."

Reply:
Yes, your wording here is much better.

Reply:
Fixed.